src/index.ts

```diff
@@ -314,7 +314,7 @@ export type {
   MongoLoggerOptions,
   SeverityLevel
 } from './mongo_logger';
-export { createStdLogger } from './mongo_logger';
+export { createStdioLogger } from './mongo_logger';
```

Comment:
This doesn't need to be exported, is this for testing? the test/mongodb.ts file imports all the things from all the files under the `src`

Reply:
npm install was complaining about this and since it's marked internal I assumed this was alright to do, but I can go ahead and fix this